resources/views/campaigns/show.blade.php

```diff
@@ -53,7 +53,7 @@
 
                                 @if ($campaign->isPublic())
                                 <dt>{{ __('campaigns.fields.followers') }}</dt>
-                                <dd>{{ \App\Facades\CampaignCache::followerCount() }}</dd>
```

Comment:
Check if CampaignCache::followerCount() is used anywhere else, and if not, you can delete the code associated to it in `app\Services\Caches\CampaignCacheService`. Clear up both `followerCount()` and `clearFollowerCount()`